pkg/features/kube_features.go

```diff
@@ -852,6 +852,16 @@ const (
 	// restore the old behavior. Please file issues if you hit issues and have to use this Feature Gate.
 	// The Feature Gate will be locked to true and then removed in +2 releases (1.35) if there are no bug reported
 	DisableCPUQuotaWithExclusiveCPUs featuregate.Feature = "DisableCPUQuotaWithExclusiveCPUs"
+
+	// owner: @munnerz
+	// kep: https://kep.k8s.io/4742
+	// alpha: v1.33
+	//
+	// Enables the PodTopologyLabelsAdmission admission plugin to automatically set node topology labels
+	// (i.e. those with the 'topology.k8s.io/' prefix on Node objects) onto Pod objects when they are
+	// bound/scheduled to a node.
```

Comment:
We should probably capture the detail here - it enables:

1) copying *all* labels from Binding objects to Pod objects during (i.e. during scheduling), *without* overwriting existing labels on the Pod

2) a mutating admission plugin that copies `topology.k8s.io/{zone,region}` and `kubernetes.io/hostname` labels onto Binding objects.

Reply:
I'm on the fence about whether we really need kubernetes.io/hostname. In most cases the hostname is the same as the node name and you can already get node name with downward API using `spec.nodeName`

Reply:
Have removed this for now - I think we should explore precisely what the use-cases for hostname are (I imagine node-local agents wanting to connect to a kubelet may be one..) as these may be better served through an alternative and more robust mechanism (e.g. `nodes/proxy` or something)